src/components/presentational/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,50 @@
+.progress-bar {
```

Comment:
Should prefix all CSS classes with .mdhui-, so this should be .mdhui-progress-bar

I would also nest all classes in this file under that - so for the fill it shuold be ```.mdhui-progress-bar .mdhui-progress-bar-fill```

Reply:
(alternative to consider at some point is using CSS modules; but so far we haven't taken that leap and CSS is still being overridden in places)